x-pack/plugins/spaces/public/management/spaces_grid/spaces_grid_page.tsx

```diff
@@ -110,6 +110,11 @@ export class SpacesGridPage extends Component<Props, State> {
         <EuiInMemoryTable
           itemId={'id'}
           items={this.state.spaces}
+          tableCaption={i18n.translate('xpack.spaces.management.spacesGridPage.tableCaption', {
+            defaultMessage: 'List of {count} spaces.',
+            values: { count: this.state.spaces.length },
+          })}
```

Comment:
Similarly to above, let's simplify this

```suggestion
          tableCaption={i18n.translate('xpack.spaces.management.spacesGridPage.tableCaption', {
            defaultMessage: 'Spaces',
          })}
```

This one is actually maybe a little to simple because the word "spaces" could mean a lot. Would calling it "Kibana Spaces" or "Security Spaces" be accurate?

Reply:
I'll change this to read `Kibana spaces` to be more descriptive. Spaces don't _require_ security, so I think "Security Spaces" wouldn't be entirely accurate